[WIP] CSV data

orel-david opened the pull request:
This is a CSV test file which check the code with random number, and the actual code that will run on the robot.
note: there is two fictive variables

pklito committed:
Add isReversed and isRelative parameters

pklito committed:
Implement isRelative

pklito committed:
Remove redundant and useless code

Removed prints of the path, and removed reset of the encoders on pure pursue initialize

pklito committed:
Implemented isReversed

orel-david committed:
create csv test file

orel-david committed:
Make csv that will put data into csv file

orel-david committed:
make a csv test file

orel-david committed:
fix coloums in csv

orel-david committed:
Fix the problem with the title

orel-david committed:
Change two variables for testing

orel-david committed:
Move the csv to a new class

orel-david committed:
Add write to csv

orel-david committed:
Add CSV to robot

orel-david committed:
Add CSV close to disabled init

katzuv committed:
Refactor class name

LiorBarkai58 committed:
Fix constant package level

LiorBarkai58 committed:
Change the package level of x and y of the point

katzuv reviewed (changes requested):
I suggest adding the current time in every recording. Also, use the WPILib Timer.

orel-david committed:
Merge remote-tracking branch 'origin/master' into CSV

orel-david committed:
Make the csv to put actual values

orel-david committed:
Add the time argument to the csv

orel-david commented:
@DanKatzuv  in the last commit i added the time parameter to the csv

LiorBarkai58 committed:
Tweak Constants

orel-david committed:
Delete CSV_test.java

Deleted the temporary class we used to check the CSV when we couldnt work on the robot

orel-david committed:
Change CSV to non-static

orel-david committed:
Merge remote-tracking branch 'origin/CSV' into CSV

LiorBarkai58 committed:
Merge pull request #56 from DanKatzuv/pure-pursue-enhancements

Merge pure-pursue-enhanements into csv

LiorBarkai58 committed:
Fix constant overlap after merge

LiorBarkai58 committed:
Improve the reset of the stringbuilder

LiorBarkai58 committed:
Improve imports

imported some libraries for changing perms of files

LiorBarkai58 committed:
Fix package level of constants and points

LiorBarkai58 committed:
Merge remote-tracking branch 'origin/CSV' into CSV

orel-david committed:
pull

orel-david committed:
Add a testing file to the csv class

orel-david committed:
Make the csv work with networktables entries

orel-david committed:
Make the a file to match entries to variables

orel-david committed:
Add the setup for the networktables
